### PR TITLE
fix: add <Mahikolhe23> 

### PR DESCRIPTION
**Pull request by Mahikolhe23**
resolves ##1806 

## Checklist for profile addition

- [x] Name the PR `fix: add <Mahikolhe23>`.
- [x] You've added file named `Mahikolhe23.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/Mahikolhe23.json
Comment (Panquesito7) [inline profiles/Mahikolhe23.json:2]:
```suggestion
  "image": "https://raw.githubusercontent.com/Mahikolhe23/Mahikolhe23/main/Assets/Mahi%20Git%20Profile.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Mahikolhe23 — COMMENTED** [review]
changes done as per requested

**Comment by Mahikolhe23** [comment]
kindly review

File: profiles/Mahikolhe23.json
Comment (Mahikolhe23) [reply profiles/Mahikolhe23.json]: done

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by Mahikolhe23** [comment]
Kindly merge 🙏

**Comment by Panquesito7** [comment]
> Kindly merge 🙏

Your pull request will be merged once one or more maintainers approve the PR.

**Comment by Mahikolhe23** [comment]
> > Kindly merge 🙏
> 
> Your pull request will be merged once one or more maintainers approve the PR.
> 

Okay

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge :tada:.

**Comment by Mahikolhe23** [comment]
Thanks

**Pull request merged**
